packages/wrangler/src/pages/build.ts

```diff
@@ -49,6 +60,10 @@ export function Options(yargs: CommonYargsArgv) {
 				type: "string",
 				description: "The location for the build metadata file",
 			},
+			"project-directory": {
+				type: "string",
+				description: "The location of the Pages project",
+			},
```

Comment:
Why add this arg, rather than just using current working directory?

Reply:
Because CI doesn't run this command in the root of the Pages project, and so needs to have a way to find it